### PR TITLE
Delete tmp files even when failure

### PR DESCRIPTION
**Pull request by dcdarrell9**
[card](https://trello.com/c/01sVkI1I)

## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by dcdarrell9** [comment]
[sdx-transform-cs tmp files do not get deleted when the transform fails](https://trello.com/c/01sVkI1I/401-sdx-transform-cs-tmp-files-do-not-get-deleted-when-the-transform-fails)

File: tests/test_transform.py [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+import csv
 import glob
-import os
 import io
+import json
+import os
+import unittest
 import zipfile
 from datetime import datetime
+from unittest.mock import patch
```
Comment (JamesGardiner) [inline tests/test_transform.py]: Alphabetized imports

File: transform/views/main.py [hunk 2]
```diff
@@ -162,12 +162,16 @@ def render_images():
...
     try:
         path = itransformer.create_pdf(survey, survey_response)
+        locn = os.path.dirname(path)
         images = list(itransformer.create_image_sequence(path))
         index = itransformer.create_image_index(images)
         zipfile = itransformer.create_zip(images, index)
-        itransformer.cleanup(os.path.dirname(path))
     except IOError as e:
         return client_error("IMAGES:Could not create zip buffer: {0}".format(repr(e)))
+    try:
```
Comment (JamesGardiner) [inline transform/views/main.py]: 1 line between try/exc blocks

File: tests/test_transform.py [hunk 3]
```diff
@@ -1,17 +1,17 @@
-import csv
+from transform import app
```
Comment (JamesGardiner) [inline tests/test_transform.py]: Import statements should be in three blocks - the first should be stdlib imports, the second installed packages, and the third should be local imports

**Pull request merged**